rero_ils/modules/holdings/cli.py

```diff
@@ -70,6 +74,42 @@ def get_random_location(org_pid):
     return get_location(next(iter(libraries or []), None))
 
 
+def create_issues_from_holding(holding):
+    """Receive randomly between 2 and 9 new issues for the given holding."""
+    for issue_number in range(1, random.randint(2, 9)):
+        # TODO: when the issue api is implemented the correct
+        # expected date will given
+        first_expected_date = holding.get(
+            'patterns').get('first_expected_date')
+        call_number = '{}_{}'.format(holding.pid, str(issue_number).zfill(5))
```

Comment:
Maybe named format parameters are better to understand:
`
'{pid}_{number}'.format(pid=holding.pid, number=str(issue_number).zfill(5))
`